configureWorkspace/configure.ts

```diff
@@ -47,6 +47,37 @@ WORKDIR /app
 EXPOSE ${port}
 COPY $source .
 ENTRYPOINT dotnet ${serviceName}.dll
+`;
+
+        case 'python':
+
+            return `
+# Python support extends back to 3.4 and 2.7 and can be specified down
+# to the micro version (e.g. 3.6.3).
+# OS Support also exists for jessie & stretch (slim and full).
+FROM python:3.6-alpine3.6
```

Comment:
Prefer `FROM python:alpine` so we always have the latest stable Python version (right now 3.6.4) by default with the latest stable Alpine.

Note that for some reason this tag currently doesn't use the latest stable Alpine version. That's a separate issue I will take up with the official Docker Python Image maintainers.

Can add a commend you can specify the exact Python and Alpine version like:
`# FROM python:3.6-alpine3.6`

I'll reach out

Probably should make sure that tag is used for newer

Reply:
FYI: Filed the issue with the `alpine` tags here: https://github.com/docker-library/python/issues/265

Reply:
Actually, why not add a comment to refer to `https://hub.docker.com/r/library/python/` for a list of all supported Python version / distribution pairs :)

Reply:
Done and done!